rafthttp/peer.go

```diff
@@ -33,6 +34,17 @@ const (
 	recvBufSize = 4096
 )
 
+type status struct {
+	name   string
+	active bool
+	*opError
+}
+
+type worker struct {
```

Comment:
It is hard to know what the work exactly is. Can you use a more specific name for the "worker"?